services/src/main/java/org/exoplatform/officeonline/AbstractOfficeOnlineService.java

```diff
@@ -480,17 +577,24 @@ public EditorConfig buildEditorConfig(String token) throws OfficeOnlineException
     }
 
     List<Permissions> permissions = new ArrayList<>();
-    List<String> values = Arrays.asList(decryptedToken.split(TOKEN_DELIMITE_SPLIT));
-    if (values.size() > 2) {
+    List<String> values = Arrays.asList(decryptedToken.split(TOKEN_DELIMITER_SPLIT));
```

Comment:
Name could be simpler: TOKEN_DELIMITER

Reply:
There are two constants:
TOKEN_DELIMITER = "+";
TOKEN_DELIMITER_SPLIT = "\\+";
We have '\\' before '+' to make regexp work as needed.

Reply:
TOKEN_DELIMITER_PATTERN then?

Reply:
Fixed: https://github.com/exo-addons/office-online/pull/15/commits/fd8d51245149a642033afd00766e6dad11dd4792